src/settings.cmake

```diff
@@ -4,6 +4,13 @@
 
 set (CMAKE_CXX_STANDARD 11)
 
+if(NOT MSVC)
```

Comment:
This doesn't look right as it would always print the warning on Unix (and PIE is Unix only concept). This whole 'if' block should go away.